src/scripts/kube-jobs/recencyToBreachId.js

```diff
@@ -0,0 +1,160 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/**
+ * Executes once
+ * Execute after `deprecateBreachResolve.js`
+ * The purpose of the script is to convert all recency indices to breach ids for `breach_resolution`
+ * For backwards compatibility, all converted `breach_resolution` fields will have a boolean
+ * `useBreachId: true/false`
+ */
+
+import Knex from 'knex'
+import knexConfig from '../../db/knexfile.js'
+import { getAllBreachesFromDb } from '../../utils/hibp.js'
+import { getAllEmailsAndBreaches } from '../../utils/breaches.js'
+const knex = Knex(knexConfig)
+
+const LIMIT = 1000 // with millions of records, we have to load a few at a time
+let subscribersArr = []
+
+const selectAndLockResolutions = async () => {
+  const trx = await knex.transaction()
+  let subscribersArr = []
+  try {
+    subscribersArr = await knex.select('id', 'primary_email', 'breach_resolution')
+      .from('subscribers')
+      .whereNotNull('breach_resolution')
+      .whereNull('db_migration_2')
+      .limit(LIMIT)
+      .orderBy('updated_at', 'desc')
+      .transacting(trx)
+      .forUpdate()
+
+    // update the lock
+    await Promise.all(subscribersArr.map(sub => {
+      const { id } = sub
+      return knex('subscribers')
+        .where('id', id)
+        .update({
+          db_migration_2: true
+        })
+        .transacting(trx)
+    }))
+
+    await trx.commit()
+  } catch (error) {
+    await trx.rollback()
+    console.log('select & mark rows failed!! first row:')
```

Comment:
Should this be `console.log()` or `console.error()`?
Minor nit, but noticed that you used `console.error()` below on L77:

https://github.com/mozilla/blurts-server/blob/28c393374ca69d9733d8605c0f7a5d56669287a1/src/scripts/kube-jobs/recencyToBreachId.js#L77

Reply:
Fixed, thanks